crypto/rand/rand_unix.c

```diff
@@ -24,6 +24,9 @@
 # include <sys/ipc.h>
 # include <sys/shm.h>
 # include <sys/utsname.h>
+# if defined(DEVRANDOM_WAIT)
+#   include "poll.h"
```

Comment:
Just a minor nit: please use `<` `>` for system headers.
and the indentation on conditional includes is just one space.
Otherwise OK.

Reply:
Fixed.